docs/connecting-data/connecting-to-apis-and-external-data/asana-api.md

```diff
@@ -1,7 +1,177 @@
----
-description: How to Connect DataDistillr to the Asana API
----
+# Connecting to the Asana API
 
-# Asana API
+## Creating an Asana account
+Set up an account with [Asana](https://asana.com/create-account).
 
-Details coming soon!
\ No newline at end of file
+### **Costs**
+There are a few different types of accounts:
+
+- Free
+- $10.99 per user per month billed annually or $13.49 billed monthly
+- $24.99 per user per month billed annually or $30.49 billed monthly
+
+You can check [HERE](https://asana.com/pricing) for details and comparison.
+
+### **Rate Limits**
+Limits are allocated per authorization token. Limits may vary from token to token.
+
+Free account limit: 150 requests per minute.
+Premium account limit: 1500 requests per minute.
+SearchAPI  are limited to 60 requests per minute.
+Duplication endpoints are limited to 5 concurrent jobs.
+
+## How to Connect DataDisillr to Asana
+To set up a data source connection for Asana, you will need to have:
+
+- A unique name for your data source connection to be used in queries.
+- The API key generated through your Asana account.
+
+### Data Source Form
+
+To locate the Asana form, follow the steps in [Connecting Your Data to DataDistillr](../../). When you get to the window to choose the data source type, select API as shown below.
+
+![Select API from the available choices] [image-3]
+
+On the API screen, select Asana from the list of API types as shown in the image below.
+
+![Select Asana from API types] [image-4]
+
+The following form will appear. Instructions can be found below on how to find the information required to fill each field on the Asana API form.
+
+Once you have filled out all the fields, press the green 'Save' button, and your API will be connected!
+
+![Asana Form] [image-1]
+
+### Name
+Enter any name that will help you recognize this data source from within your query window.
+
+Acceptable characters include:
+
+* lowercase alphanumeric characters
+* underscores
+
+### API Key
+Once you have created a workspace in your Asana account, you can generate the API key that you'll need in order to fill in the form.
+
+Click on your account icon at the top right of your screen. Choose 'My Settings'.
```

Comment:
For consistency, create a graphic for this section like is done in the JIRA form